pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -1807,6 +1807,7 @@ public class Foo {
 
     <rule name="UnnecessaryLocalBeforeReturn"
           language="java"
+          deprecated="true"
```

Comment:
Please also update the rule description of this now deprecated rule:

```
Avoid the creation of unnecessary local variables

This rule has been deprecated since 7.17.0. Use the new rule
{%rule VariableCanBeInlined %} instead, which additionally covers throw statements.
```